modules/catalog/src/main/java/org/apache/ignite/internal/catalog/UpdateContext.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.catalog;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Function;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Context contains the catalog to be updated.
+ * It is used by {@link BulkUpdateProducer} when executing a batch of catalog commands
```

Comment:
```suggestion
 * Context contains the catalog to be updated.
 *
 * <p>It is used by {@link BulkUpdateProducer} when executing a batch of catalog commands
```

Besides, javadoc is not quite correct. It states context is used in BulkUpdateProducer, but in fact it used for every update producer